scout/server/blueprints/variant/controllers.py

```diff
@@ -442,8 +442,6 @@ def get_loqusdb_obs_cases(
     obs_cases = []
     user_institutes_ids = set([inst["_id"] for inst in user_institutes(store, current_user)])
     for i, case_id in enumerate(obs_families):
-        if len(obs_cases) == 10:
```

Comment:
This could be a bit dangerous.. what if you have 5000 observations?

Reply:
Hmm, I see! Yes, you might be right. Let's bring back a slightly higher fixed max number.